src/pages/profile/ProfileTab.tsx

```diff
@@ -58,16 +56,16 @@ const ProfileTab: React.FC = () => {
               <h4>{user.email}</h4>
             </IonItem>
             <IonItem lines="none">
-              <IonButton
-                size="default"
-                onClick={() => setIsReadOnly(!isReadOnly)}
-              >
-                Edit Profile
-              </IonButton>
-              <IonButton
-                size="default" 
-                onClick={logoutUser}>
-                   logout
+              <div className="editButton">
+                <IonButton
```

Comment:
Du kannst statt die Components nochmal in `<div>` tags zu wrappen, die class auch auf die Ionic Componenten anwenden. Dort heisst das attribut allerdings nicht 'className' sondern nur 'class'. Also: `<IonButton class="editButton">.` Dann kann das `<div>` weg und wir sparen code und ggf unnötiges spacing :)